requirements/base.txt

```diff
@@ -19,3 +19,4 @@ scikit-learn==0.24.*
 Shapely>=1.7.1,<=1.8.0
 hpopt==0.3.0
 imagesize==1.4.1
+psutil
```

Comment:
BSD 3-Clause "New" or "Revised" License.
https://github.com/giampaolo/psutil/blob/master/LICENSE

Looks OK but @yunchu could you check this?

Reply:
@yunchu It's a reminder. Could you please confirm when you are available?

Reply:
@goodsong81 @eunwoosh 
There is no issue to use BSD 3-Clause "New" or "Revised" License. 
The only thing required is add this dependency to the BOM to update OSPDT for the next release.